----------------------------------------------------------------
Fri Jun 05 19:54:28 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a816c00e-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:01:33 BRT 2020:
Shutting down instance a816c00e-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:01:34 BRT 2020 Thread[Thread-204,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:01:34 BRT 2020 Thread[Thread-204,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:01:35 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 5d3080f5-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:02:08 BRT 2020:
Shutting down instance 5d3080f5-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:02:08 BRT 2020 Thread[Thread-403,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:02:08 BRT 2020 Thread[Thread-403,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:02:09 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 3f3e811c-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:07:47 BRT 2020:
Shutting down instance 3f3e811c-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:07:47 BRT 2020 Thread[Thread-602,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:07:47 BRT 2020 Thread[Thread-602,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:07:48 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 9188413f-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:07:55 BRT 2020:
Shutting down instance 9188413f-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:07:55 BRT 2020 Thread[Thread-801,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:07:55 BRT 2020 Thread[Thread-801,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:07:56 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 52f6815c-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:10:43 BRT 2020:
Shutting down instance 52f6815c-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:10:43 BRT 2020 Thread[Thread-1000,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:10:43 BRT 2020 Thread[Thread-1000,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:10:44 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 45370179-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:11:41 BRT 2020:
Shutting down instance 45370179-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:11:41 BRT 2020 Thread[Thread-1199,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:11:41 BRT 2020 Thread[Thread-1199,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:11:42 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance b849c196-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:11:54 BRT 2020:
Shutting down instance b849c196-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:11:54 BRT 2020 Thread[Thread-1398,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:11:54 BRT 2020 Thread[Thread-1398,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:11:55 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance fc2ec1b3-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:12:12 BRT 2020:
Shutting down instance fc2ec1b3-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:12:13 BRT 2020 Thread[Thread-1597,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:12:13 BRT 2020 Thread[Thread-1597,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:12:13 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 60e601d0-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:18:15 BRT 2020:
Shutting down instance 60e601d0-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:18:16 BRT 2020 Thread[Thread-1796,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:18:16 BRT 2020 Thread[Thread-1796,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:18:16 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance cc830228-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:20:02 BRT 2020:
Shutting down instance cc830228-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:20:03 BRT 2020 Thread[Thread-1995,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:20:03 BRT 2020 Thread[Thread-1995,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:20:03 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 8ee74257-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:23:43 BRT 2020:
Shutting down instance 8ee74257-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:23:43 BRT 2020 Thread[Thread-2194,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:23:43 BRT 2020 Thread[Thread-2194,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:23:44 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 8ef50298-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:24:28 BRT 2020:
Shutting down instance 8ef50298-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:24:28 BRT 2020 Thread[Thread-2393,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:24:28 BRT 2020 Thread[Thread-2393,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:24:29 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 5da70308-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:25:41 BRT 2020:
Shutting down instance 5da70308-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:25:41 BRT 2020 Thread[Thread-2592,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:25:41 BRT 2020 Thread[Thread-2592,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:25:42 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance e4140360-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:26:03 BRT 2020:
Shutting down instance e4140360-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:26:04 BRT 2020 Thread[Thread-2791,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:26:04 BRT 2020 Thread[Thread-2791,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:26:05 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 9f764392-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:46:25 BRT 2020:
Shutting down instance 9f764392-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:46:25 BRT 2020 Thread[Thread-2990,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:46:25 BRT 2020 Thread[Thread-2990,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:46:26 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 0c6ac5ee-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:47:27 BRT 2020:
Shutting down instance 0c6ac5ee-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:47:28 BRT 2020 Thread[Thread-3189,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:47:28 BRT 2020 Thread[Thread-3189,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:47:28 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 27bc0620-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:47:41 BRT 2020:
Shutting down instance 27bc0620-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:47:41 BRT 2020 Thread[Thread-3388,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:47:41 BRT 2020 Thread[Thread-3388,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:47:42 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 58a78654-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:47:50 BRT 2020:
Shutting down instance 58a78654-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:47:50 BRT 2020 Thread[Thread-3587,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:47:50 BRT 2020 Thread[Thread-3587,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:47:51 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance aeee0671-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:48:36 BRT 2020:
Shutting down instance aeee0671-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:48:36 BRT 2020 Thread[Thread-3786,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:48:36 BRT 2020 Thread[Thread-3786,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:48:37 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 0606c68e-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:48:46 BRT 2020:
Shutting down instance 0606c68e-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:48:46 BRT 2020 Thread[Thread-3985,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:48:46 BRT 2020 Thread[Thread-3985,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:48:47 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 820346b7-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:48:51 BRT 2020:
Shutting down instance 820346b7-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:48:52 BRT 2020 Thread[Thread-4184,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:48:52 BRT 2020 Thread[Thread-4184,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:48:52 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance c68586e4-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 20:50:26 BRT 2020:
Shutting down instance c68586e4-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 20:50:26 BRT 2020 Thread[Thread-4383,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 20:50:26 BRT 2020 Thread[Thread-4383,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 20:50:27 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 14f1471a-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 21:16:12 BRT 2020:
Shutting down instance 14f1471a-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 21:16:12 BRT 2020 Thread[Thread-4582,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 21:16:12 BRT 2020 Thread[Thread-4582,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 21:16:13 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance f4354787-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 21:16:24 BRT 2020:
Shutting down instance f4354787-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 21:16:25 BRT 2020 Thread[Thread-4781,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 21:16:25 BRT 2020 Thread[Thread-4781,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 21:16:25 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 232d87a4-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 21:17:11 BRT 2020:
Shutting down instance 232d87a4-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 21:17:12 BRT 2020 Thread[Thread-4980,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 21:17:12 BRT 2020 Thread[Thread-4980,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 21:17:12 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 02f807c1-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 21:18:03 BRT 2020:
Shutting down instance 02f807c1-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 21:18:03 BRT 2020 Thread[Thread-5179,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 21:18:03 BRT 2020 Thread[Thread-5179,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 21:18:04 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance e394c7de-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 21:18:45 BRT 2020:
Shutting down instance e394c7de-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 21:18:45 BRT 2020 Thread[Thread-5378,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 21:18:45 BRT 2020 Thread[Thread-5378,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 21:18:46 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 1503c7fb-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 21:19:44 BRT 2020:
Shutting down instance 1503c7fb-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 21:19:44 BRT 2020 Thread[Thread-5577,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 21:19:44 BRT 2020 Thread[Thread-5577,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 21:19:45 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance e7450818-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 05 21:19:50 BRT 2020:
Shutting down instance e7450818-0172-86b1-54f9-0000078691a0 on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Jun 05 21:19:50 BRT 2020 Thread[Thread-5776,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Jun 05 21:19:50 BRT 2020 Thread[Thread-5776,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Jun 05 21:19:51 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance aa588835-0172-86b1-54f9-0000078691a0 
on database directory C:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/jeanb/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=c:\Users\jeanb\Desktop\facul\POO2\ProjetoFinal\Projeto-Final-em-Dupla--Gustavo-e-Jean\Projeto-Final-em-Dupla--Gustavo-e-Jean
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
